### DISCUSSION TITLE
Codespaces not working well

### DISCUSSION THREAD

**Discussion by olilarkin** [community/community#3694] [Codespaces]
A few months ago, codespaces was working really well for my project https://github.com/iPlug2/iPlug2OOS
I've been playing with it again, and now it seems unusable. Keeps saying reconnecting, and doesn't find my build tasks. The same things works well in VSCode remote containers.

**Comment by PranamBhat** [comment]
Hi @olilarkin 

Can you elaborate with exact error message.

**Reply by olilarkin** [reply]
The main thing is just a lot of latency. "Reconnecting to codespaces" messages, and sometimes not finding my build tasks

**Reply by ghost** [reply]
what is the problem ?

**Reply by olilarkin** [reply]
The performance was much better a few months ago. There is no specific error message

**Reply by olilarkin** [reply]
Just tried it and this evening performance is really good. Not sure what was happening before.

**Reply by matthewisabel** [reply]
Thanks for reporting back. We'll circle back and investigate if it happens again so please let us know.
